Workade/Views&ViewModels/Home/Constants.swift

```diff
@@ -11,7 +11,9 @@ import Foundation
 ///
 /// 실수 방지를 위해 따로 빼봤는데 현재는 homeURL만 있습니다.
 struct Constants {
-    static let homeURL = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Home/"
+    static let officeResourceAddress = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Home/office.json"
+    static let magazineResourceAddress = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Home/magazine.json"
+    static let checkListResourceAddress = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Checklist/checkList.json"
     
     static let wishMagazine = "wishMagazine"
```

Comment:
[P4] Constants에 있는 다른 정적 프로퍼티의 값이 URL 주소값 String인데 반해 wishMagazine만 홀로 UserDefault 키값이라 어색한 감이 있는 것 같은데 이를 Constants 내부의 또 다른 struct나 enum으로 분리해두는 건 어떻게 생각하시나요?

Reply:
[P4] 아! 여기에 대한 마무리가 덜됐었네요 ㅠㅠ 일단 어디로든 한 군데에 뭉치자는 생각으로 넣어두긴했는데 저런 류의 고정값들을 관리할 법에 대해 생각해보겠습니다! 저 url문자열 중에서도 중복되는 https://raw.githubusercontent.com/IIION/WorkadeData/main/ 구간은 baseURL로 빼야겠다는 생각도 드네용!

Reply:
내부에서 enum으로 분리하는걸로 수정했습니다~!
```suggestion
    struct Constants {
    enum Address {
        static let officeResource = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Home/office.json"
        static let magazineResource = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Home/magazine.json"
        static let checkListResource = "https://raw.githubusercontent.com/IIION/WorkadeData/main/Checklist/checkList.json"
    }
    
    enum Key {
        static let wishMagazine = "wishMagazine"
    }
}
```

Reply:
baseURL을 활용하는 방식은 추후 NetworkManager의 싱글톤을 Moya처럼 프로토콜 형태로 분리할 예정인데, 그 때가서 반영해보겠습니다~!